packages/google_maps_flutter/README.md

```diff
@@ -24,7 +24,18 @@ To use this plugin, add `google_maps_flutter` as a [dependency in your pubspec.y
 
 ## Getting Started
 
-Get an API key at <https://cloud.google.com/maps-platform/>.
+* Get an API key at <https://cloud.google.com/maps-platform/>.
+
+* Enable Google Map SDK for each platform.
+  * Go to [Google Developers Console](https://console.cloud.google.com/).
+  * Choose the project that you want to enable Google Maps on.
+  * Select the navigation menu and then select "Google Maps".
+  * Select "APIs" under the Google Maps menu.
+  * To enable Google Map for Android, select "Maps SDK for Android" in the "Additional APIs" section, then select "ENABLE".
+  * To enable Google Map for iOS, select "Maps SDK for iOS" in the "Additional APIs" section, then select "ENABLE".
+  * Make sure the APIs you enabled are under the "Enabled APIs" section.
```

Comment:
This has a potential to be a maintenance burden if the GDC undergoes a big update, which I doubt would happen every frequently. Should we mention that this steps might be out of date? cc @amirh